com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs

```diff
@@ -97,8 +122,80 @@ public override void OnInspectorGUI()
             }
 
             EditorGUI.EndDisabledGroup();
+            if (!serializedObject.isEditingMultipleObjects)
+            {
+                DrawStreamSettings(streamPlayer);
+            }
+
+            serializedObject.ApplyModifiedProperties();
+        }
+
+        static void DrawStreamSettings(AlembicStreamPlayer player)
```

Comment:
Can any of that be shared with the ScriptedImporter settings?

Reply:
I was a bit sloppy here I think:
In the best case I would make a PropertyDrawer for AlembicStreamSettings. Will how "generic" this draws is (might be a different way of drawing based on context)